packages/grafana-data/src/dataframe/ArrowDataFrame.ts

```diff
@@ -91,6 +91,17 @@ export function arrowTableToDataFrame(table: Table): ArrowDataFrame {
   };
 }
 
+// fieldNamePrefixSep is the delimiter used with fieldNamePrefix.
+const fieldNamePrefixSep = '🦥: ';
```

Comment:
This matches:
https://github.com/grafana/grafana-plugin-sdk-go/blob/df_field_name_prefix/data/arrow.go#L76

Reply:
Note also that the updated all_types.golden.arrow includes names with the akward prefix seperator